src/screens/poa-form/index.js

```diff
@@ -426,6 +427,56 @@ class PoAForm extends React.Component {
       <Section>
         {disclaimer}
         <Heading tag='h1'>POWER OF ATTORNEY FOR CARE OF A MINOR CHILD</Heading>
+        <div>
+
+        <Stepper 
+          steps={ 
+            [
+              {
+                title: 'Child Information',
+                onClick: (e) => {
+                  e.preventDefault();
+                  this.renderStepOne();
```

Comment:
Setting the state to `{ step: 0 }` will automatically trigger `renderForm`, so you don't need to call `renderStepOne` in these.